CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Please also have a look at our
 
 ### Changed
 
+- Mark `OutputFormat` as not extendable and the constructor as `@internal`
+  (#1131) 
```

Comment:
If my suggestion for the class DocBlock is accepted, this probably warrants a second changelog entry in the 'Deprecated' section.  (But I don't think two separate PRs are needed!)